dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -262,7 +260,12 @@ public Response deleteType(@PathParam("idOrVar") final String idOrVar, @Context
 	@JSONP
 	@NoCache
 	@Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-	public Response getType(@PathParam("idOrVar") final String idOrVar, @Context final HttpServletRequest req, @Context final HttpServletResponse res)
+	public Response getType(
+			@PathParam("idOrVar") final String idOrVar,
+			@Context final HttpServletRequest req,
+			@Context final HttpServletResponse res,
+			@QueryParam("language_id") final Long languageId,
```

Comment:
In order to follow the same naming pattern of the method we could change the parameter to `languageId`